src/containers/AnnotationsPane.jsx

```diff
@@ -0,0 +1,372 @@
+/*
+Annotations Pane
+----------------
+
+TODO: Description
+
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Utility } from '../lib/Utility';
+
+import {
+  setRotation, setScaling, setTranslation, resetView,
+  setViewerState, updateViewerSize, updateImageSize,
+  SUBJECTVIEWER_STATE,
+} from '../ducks/subject-viewer';
+
+const INPUT_STATE = {
+  IDLE: 0,
+  ACTIVE: 1,
+}
+
+//Add ?dev=1 to the URL to enable DEV_MODE
+const DEV_MODE = window.location && /(\?|&)dev(=|&|$)/ig.test(window.location.search);
+
+class SubjectViewer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    //HTML element refs.
+    this.section = null;
+    this.svg = null;
+    this.svgImage = null;
+
+    //Events!
+    this.updateSize = this.updateSize.bind(this);
+    this.onImageLoad = this.onImageLoad.bind(this);
+    this.onMouseDown = this.onMouseDown.bind(this);
+    this.onMouseUp = this.onMouseUp.bind(this);
+    this.onMouseMove = this.onMouseMove.bind(this);
+    this.onMouseLeave = this.onMouseLeave.bind(this);
+    this.onWheel = this.onWheel.bind(this);
```

Comment:
This isn't user by the `AnnotationsPane` component, and the function was recently removed from the `SubjectViewer`.